### ISSUE TITLE
InfluxDB : Templating Cusotm All Value should not be formated / escaped

### ISSUE THREAD

**Issue by xlash** [grafana/grafana#4736]
I have added a few templating variables to use with my InfluxDB data source. I see that grafana escapes my all characters in a way not allowed by InfluxDB. i cannot see if this has already been identified or workaround.

Example:

`SELECT` count(v) FROM db."default".table WHERE var1=~ /^val1$/ AND var2=~ /^$/ AND "var3" =~ /^\*$/ AND time > now() - 6h GROUP BY `time(15s)`

I tried the following combinations without success :

Grafana All Value           |         InfluxDB Query received:    |     Result
---------------------------     |    -----------------------------------    |   ----------------
None                              |                    /^$/                            |        Fail
*                                     |                    /^\\*$/                          |        Fail
.+                                   |                     /^\\.\\+$/                      |         Fail

Removing the escaped characters and trying it out in InfluxDB give me the correct answer on Test 2 and 3 above.

Thanks

**Comment by torkelo** [comment]
custom all value should not be escaped I think, try using the automatic all value, should work as a temp work around

**Comment by xlash** [comment]
That is correct. the Default value pick all the options and is working correctly. (I think I missed it due to another query issue earlier.

Thanks

**Comment by bbczeuz** [comment]
Similar issue here:

* What grafana version are you using? 3.0.0-beta5, commit: v3.0-beta4+14-gd98839f, build date: 2016-04-15 15:01:27
* What datasource are you using? InfluxDB 0.12.1
* What OS are you running grafana on? RHEL 7.x

While the multiple query issue was fixed in #4533, there seems to be another issue with the updates versions of InfluxDB+Grafana:

Query:

```
SELECT ($adc_offset+mean(value))*$adc_gain FROM "adc" WHERE $timeFilter GROUP BY time($interval), "host", "chan"
```
With the two template values $adc_offset = -59000 and $adc_gain = 0.00174216


Grafana escapes the query to
```
SELECT (-59000+mean(value))*0\.00174216 FROM "adc" WHERE time > now() - 1m GROUP BY time(100ms), "host", "chan"
```
Bug at: ```0\.00174216```

So InfluxDB complains:
```
{
  "error": "error parsing query: found \\, expected FROM at line 1, char 32",
  "message": "error parsing query: found \\, expected FROM at line 1, char 32"
}
```

If I manually put the numbers into the queries, everything works as expected.

**Comment by torkelo** [comment]
fixed :)
